packages/connect-popup/e2e/tests/permissions.test.ts

```diff
@@ -18,23 +19,49 @@ const fixtures = [
     {
         description: `iframe and host same origins but with trust-issues=true -> show permissions`,
         queryString: '?trust-issues=true',
+        setTrustedHost: false,
         expect: () => popup.waitForSelector("[data-test='@permissions/confirm-button']"),
     },
     {
         description: `iframe and host same origins but with trust-issues=false -> no permissions shown`,
         queryString: '?trust-issues=false',
+        setTrustedHost: false,
         expect: () =>
             popup.waitForSelector('//p[contains(., "Follow instructions on your device")]', {
                 state: 'visible',
                 strict: false,
             }),
     },
+    {
+        description:
+            'iframe and host same origins but with settings to trustedHost -> no permissions shown',
```

Comment:
I might seem retarded, but please once again - we need this because we can employ query string for webextension right? 

just summarizing my thoughts: 
- normally, sldev, localhost, etc are trusted by default. 
- we may downgrade their trust level (useful for tests) either by query string param, or now, newly by init param.
- webextension, on the other hand, does not run on a trusted domain. so, by default it displays permissions screen etc. here you are introducing an option to increase trust level? my question is (and maybe it was already answered elsewhere): how do we guarantee that this is not abused by a 3rd party?

Reply:
Yes, we cannot use query string in web extension so we need other way to simulate the not trusted environment even iframe and popup are running in same domain during tests `http://localhost:8088`

Your points are almost right, but the last one. webextension is running on TRUSTED domain since iframe and popup are running under same domain, but we want to test the scenario where it is not trusted domain and for that we need to do it throw TrezorConnect.init becuase the query param does not work in webextension.

I think it is a mess to have 2 ways of doing the same so I would use only one, the settings page, and maybe we can run all the tests with 2 flows (trusted and not trusted) so we test both flows for every test because now for some tests we are testing with trusted host and other test we are using not trusted hosts.

Reply:
> webextension is running on TRUSTED domain since iframe and popup are running under same domain, 

Iframe and popup are always running on the same domain. so this point is not valid. I still don't understand how webextension is percieved as trusted here? it must be something specific to testing setup?

Reply:
its about host vs iframe domain. and since extension is never sldev nor trezor.io (or localhost) it imho can never be trusted.

Reply:
Sorry I mistook it , I wanted to say host instead of popup.

But I found the reason, it is because iframe is running on localhost during the tests then it is consider trusted even the extension is not whitelisted.

https://github.com/trezor/trezor-suite/blob/develop/packages/connect-iframe/src/connectSettings.ts#L44